solidity-checks/src/main/java/org/sonarsource/solidity/checks/CheckVerifier.java

```diff
@@ -29,8 +29,9 @@ private CheckVerifier(IssuableVisitor checkVisitor, String relativePath, boolean
       checkVisitor.setRuleContext(testRuleContext);
       checkVisitor.visit(parser.sourceUnit());
 
+      parser.comments.forEach(x -> System.out.println(x.getText()));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=33&issues=AWRuzWDlI66nRh8YCmzm&open=AWRuzWDlI66nRh8YCmzm)

[ISSUE_KEY]: AWRuzWDlI66nRh8YCmzm